contrib/scripts/codepage_to_utf8.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
```

Comment:
Suggest: `#!/usr/bin/env python3`

This an improvement over our previous python scripts in the repo, which are still using `#!/usr/bin/python3`, which isn't as compatible for virtual environments. 

More info here: https://discuss.python.org/t/pep-394-clarify-bullet-for-usr-bin-env-python/7953/2

> *such a clarification would also benefit PEP 397 as the Python Launcher admits “virtual” shebangs, so it will provide further impetus for Windows users to add `#!/usr/bin/env python3` to their scripts to ensure the greatest degree of cross-platform compatability between Linux, MacOS, and Windows.*